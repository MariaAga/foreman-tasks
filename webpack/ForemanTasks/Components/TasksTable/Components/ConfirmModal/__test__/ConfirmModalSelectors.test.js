import { testSelectorsSnapshotWithFixtures } from '@theforeman/test';
import {
  selectActionText,
  selectActionState,
  selectActionType,
} from '../ConfirmModalSelectors';

const state = {
  foremanTasks: {
    confirmModal: {
      actionText: 'some-text',
      actionState: 'some-state',
      actionType: 'some-type',
    },
  },
};

const fixtures = {
  'should select actionText': () => selectActionText(state),
  'should select actionState': () => selectActionState(state),
  'should select actionType': () => selectActionType(state),
};

describe('TasksDashboard - Selectors', () =>
  testSelectorsSnapshotWithFixtures(fixtures));
