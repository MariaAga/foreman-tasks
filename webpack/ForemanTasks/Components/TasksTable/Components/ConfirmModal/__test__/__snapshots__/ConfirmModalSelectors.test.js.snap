// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TasksDashboard - Selectors should select actionState 1`] = `"some-state"`;

exports[`TasksDashboard - Selectors should select actionText 1`] = `"some-text"`;

exports[`TasksDashboard - Selectors should select actionType 1`] = `"some-type"`;
